src/components/Lib/web3-utils.js

```diff
@@ -0,0 +1,13 @@
+import { toChecksumAddress } from 'web3-utils'
```

Comment:
@rperez89 I recently used [this approach](https://github.com/aragon/aragon-ui/pull/659) to avoid importing web3-utils in aragonUI: do you think it would work here? Or are we going to need more things from web3-utils?

Reply:
mmm not sure if we are going to be using more utils, what do you think?

Reply:
Not sure, let’s see if we start using other things, and we’ll move back to `web3-utils` at this point!